core/src/main/java/com/onelogin/saml2/util/Util.java

```diff
@@ -550,191 +551,40 @@ public static String convertToPem(X509Certificate certificate) {
 			LOGGER.debug("Error converting certificate on PEM format: "+ e.getMessage(), e);
 		}
 		return pemCert;
-	}	
-
-	/**
-	 * Redirect to location url
-	 * 
-	 * @param response
-	 * 				HttpServletResponse object to be used
-	 * @param location
-	 * 				target location url
-	 * @param parameters
-	 * 				GET parameters to be added
-	 *
-	 * @throws IOException
-	 *
-	 * @see javax.servlet.http.HttpServletResponse#sendRedirect(String)
-	 */
-	public static void sendRedirect(HttpServletResponse response, String location, Map<String, String> parameters) throws IOException {
```

Comment:
`Util` no longer depends on `javax.servlet`, those methods are now in `ServletUtils`.